app/assets/stylesheets/application.css.scss

```diff
@@ -87,6 +87,20 @@ body {
   background-position: center center;
 }
 
+.application-body-restart-true{
+  background-image: url('restart-true.png');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-restart-false{
+  background-image: url('restart-false.jpg');
+  background-size:     cover;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size<br>Colon after property should be followed by one space